### PR TITLE
edit styles and edit detailClass

### PR DESCRIPTION
**Pull request by nhanlinh1478**

### CODE REVIEW

File: src/components/classroom/classroomsSlice.js [hunk 1]
```diff
@@ -17,7 +18,14 @@ export const createClassroom = createAsyncThunk(
...
 )
-
+// export const getClassroom = createAsyncThunk(
+//   'classrooms/classroom/:id',
+//   async (classroom) => {
+//     const response = await axiosClient.get('api/classrooms/:id')
+//     console.log(response.data)
+//     return response.data
+//   }
+// )
```
Comment (nguyen315) [inline src/components/classroom/classroomsSlice.js]: Remove comment code

File: src/components/pages/DetailClassroom.js [hunk 2]
```diff
@@ -1,63 +1,130 @@
...
-  createData('Ice cream sandwich', 237, 9.0, 37, 4.3),
-  createData('Eclair', 262, 16.0, 24, 6.0),
-  createData('Cupcake', 305, 3.7, 67, 4.3),
-  createData('Gingerbread', 356, 16.0, 49, 3.9),
-]
+  let { id } = useParams()
+  useEffect(() => {
+    async function fetchData() {
+      const results = await axiosClient.get(`/api/classrooms/${id}`)
+      console.log(results.data)
```
Comment (nguyen315) [inline src/components/pages/DetailClassroom.js]: remove console.log

File: src/components/pages/DetailClassroom.js [hunk 3]
```diff
@@ -1,63 +1,130 @@
...
-  const { id } = useParams()
-  console.log(id)
+      setClassroom(results.data)
+    }
+
+    fetchData()
+  }, [])
+  const handleCopyLink = () => {
+    console.log(copyLink)
+  }
```
Comment (nguyen315) [inline src/components/pages/DetailClassroom.js]: unuse function, please remove

**Review by nguyen315 — COMMENTED** [review]
@nhanlinh1478

**Comment by nhanlinh1478** [comment]
ok

**Pull request merged**